### DISCUSSION TITLE
iTwin to Unity 4d Sequencing

### DISCUSSION THREAD

**Discussion by VescanAlexandru** [iTwin/itwinjs-core#4570] [Q&A] [answered]
Good afternoon all,
Is there any route to stream the iModel 4d sequence in to Unity using the iTwin.js?

**Comment by MichaelBelousov** [comment, marked as answer]
I assume you are already aware of [our iTwin.js with Unity sample](https://github.com/iTwin/imodel-unity-example). 

It does not currently support [schedule script](https://www.itwinjs.org/reference/core-common/displaystyles/renderschedule/renderschedule.script/) playback.

**Reply by VescanAlexandru** [reply]
Thank you very much for your answer!

**Reply by MichaelBelousov** [reply]
If you would like to ask further questions on how various features of schedule scripts might be implemented in unity, please feel free to start another discussion. Some items, such as growth animations, could be difficult to implement in unity. Others like simple transform and visibility animations should be much simpler.

**Comment by MichaelBelousov** [comment]
For other people looking at this, here's a sample approach that was made: https://github.com/mgooding/itwin-render-timeline-unity
